docker/espnet.devel

```diff
@@ -0,0 +1,57 @@
+FROM nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04
+
+ENV CUDA_HOME /usr/local/cuda
+RUN apt-get update && apt-get install -y \
+        cmake \
+        wget \
+        build-essential \
+        curl \
+        git \
+        libfreetype6-dev \
+        libpng12-dev \
+        libzmq3-dev \
+        pkg-config \
+        python-dev \
+        software-properties-common \
+        swig \
+        zip \
+        zlib1g-dev \
+        libcurl3-dev \
+        libhdf5-dev \  
+        libsndfile-dev \
+        libasound2-dev \
+        python-numpy-dev \ 
+        python-tk \
+        pciutils \
+        libatlas3-base \
+        automake \
+        autoconf \
+        libtool \
+        subversion \
+        flac \
+        bc \
+        gawk \
+        sox \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN curl -fSsL -O https://bootstrap.pypa.io/get-pip.py && \
+    python get-pip.py && \
+    rm get-pip.py
+
+RUN pip install numpy ipython \
+                h5py scipy \
+                matplotlib scikit-image \
+                scikit-learn colorama virtualenv
+
+RUN mkdir -p /espnet/tools
```

Comment:
I'm not sure why you copy them.
I thought that we can still use the original directory structure.

Reply:
I copy to speed up the building process. If there is additional information inside one egs folder, it will take time to buffer and build. Also, to isolate the tools built from the container.